### PR TITLE
test(#45): Add application load tests

### PR DESCRIPTION
**Pull request by migromarj**
Add 'locust' tool as project requirement.
Add tests for the home page, login page, registration page, user profile and event details in loadtest/locustfile.py.

### CODE REVIEW

**Review by juradosalado — APPROVED** [review]
All tests run with no failures. Good job!

File: loadtest/locustfile.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    host = HOST
+    tasks = [DefLogin]
+    wait_time = between(3,5)
+
+class Register(HttpUser):
+    host = HOST
+    tasks = [DefRegister]
+    wait_time = between(3,5)
+
+class ParticipantProfile(HttpUser):
```
Comment (TomasCB24) [inline loadtest/locustfile.py:64]: This test doesn't work

**Review by TomasCB24 — CHANGES_REQUESTED** [review]
The ParticipantPerfil test doesn´t work.

File: loadtest/locustfile.py (hunk 1 above)
Comment (migromarj) [reply loadtest/locustfile.py:64]: True, I just saw that it is an error that is also in the main branch, I solve it and upload the changes

**Review by TomasCB24 — APPROVED** [review]
Now works. 💯.

**Review by juradosalado — APPROVED** [review]
Error fixed. Everything works all right with loaded data.

**Pull request merged**